docs/src/env-vars.md

```diff
@@ -4,7 +4,7 @@ You can also configure Martin using environment variables, but only if the confi
 
 | Environment var <br/> Config File key    | Example                              | Description                                                                                                                                                                                                |
 |------------------------------------------|--------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `DATABASE_URL` <br/> `connection_string` | `postgresql://postgres@localhost/db` | Postgres database connection                                                                                                                                                                               |
+| `DATABASE_URL` <br/> `connection_string` | `postgresql://`<br/>`postgres@localhost/db` | Postgres database connection                                                                                                                                                                               |
```

Comment:
Let's not add `<br/> here? As someone would want to copy and modify it.

Reply:
I tried - you can still copy it. The table does look a bit better this way.  We do want to reformat the markdown though